src/elements/Button/index.types.ts

```diff
@@ -0,0 +1,7 @@
+import { ColorPalette } from '@util/colorSchemes';
+
+export type ButtonStyle = {
```

Comment:
Confused about that interface...also it takes much time to find how to use button component, maybe we should add a section of usage example in PR when the PR is about a new component?

Reply:
Apologies... I think that's a good idea. I'll add a usage example in the PR write-up

Reply:
Example and write-up added! I've also pushed a commit to fix that closed modal styling mistake.